### PR TITLE
Create HACKING.md file

### PR DESCRIPTION
**Pull request by bartwensley**
Create a very basic HACKING.md file with instructions for building the NTO and running very basic tests. The intent is that this will
be a starting point for others to add content in the future. I based this on the HACKING.md file from openshift/machine-config-operator.

I know this isn't much, but thought it would be good to at least get started.

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+```
+# Specify the registry
+REGISTRY=quay.io
+# Specify the registry user
+ORG=username
+# Specify the tag
+TAG=tag
+
+make local-image-push IMAGE=${REGISTRY}/${ORG}/cluster-node-tuning-operator:${TAG}
+```
```
Comment (marioferh) [inline HACKING.md:70]: Maybe add how to install it:
`    Launch scale: oc scale deploy/cluster-version-operator --replicas=0 -n openshift-cluster-version
    Apply manifest: oc apply -f manifest/

Now custom NTO image is installed, continue with PAO deployment.`

**Review by marioferh — COMMENTED** [review]
so useful

File: HACKING.md (hunk 1 above)
Comment (bartwensley) [reply HACKING.md:70]: Thanks Mario. I'd rather not add things to the file that I can't test myself and I don't have an environment to test that in at the moment. Would it be OK if I left it for you to add this after the file is created?

Comment (marioferh) [reply HACKING.md:70]: Of course.

**Review by marioferh — APPROVED** [review]
/lgtm

**Comment by bartwensley** [comment]
@swatisehgal and @kpouget - just wondering if you could take a quick look and then merge this. Thanks!!!

**Pull request merged**